fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -296,6 +309,87 @@ public DataApiRequest(
     }
 
     /**
+     * To check whether dateTime column request is first one in the sort list or not.
+     *
+     * @param sortColumns  map of columns and its direction
+     *
+     * @return True if dateTime column is first one in the sort list. False otherwise
+     */
+    protected Boolean isDateTimeFirstSortField(Map<String, SortDirection> sortColumns) {
+        if (sortColumns != null) {
+            List<String> columns = new ArrayList(sortColumns.keySet());
+            return columns.get(0).equals(DATE_TIME_STRING);
+        }
+        else {
+            return false;
+        }
+    }
+
+    /**
+     * Method to remove the dateTime column from map of columns and its direction.
+     *
+     * @param sortColumns  map of columns and its direction
+     *
+     * @return  Map of columns and its direction without dateTime sort column
+     */
+    protected Map<String, SortDirection> removeDateTimeSortColumn(Map<String, SortDirection> sortColumns) {
+        if (sortColumns != null && sortColumns.containsKey(DATE_TIME_STRING)) {
+            sortColumns.remove(DATE_TIME_STRING);
+            return sortColumns;
+        }
+        else {
+            return sortColumns;
+        }
+    }
+
+    /**
+     * Method to generate DateTime sort column from the map of columns and its direction.
```

Comment:
Note: Having "Method to" in a javadoc on a method is not useful info, since it's clear from context (ie. the comment is on a method) that the doc is describing the method. Obviously not a _big_ issue, but it's always good to be as concise as possible 😄 

/NotABigDeal